### ISSUE TITLE
Tiến độ tiến hành chương Optimization Algorithms

### ISSUE THREAD

**Issue by duythanhvn** [aivivn/d2l-vn#1761]
Mình hy vọng chương **Optimization Algorithms** này (90 phần) chúng ta có thể hoàn thiện trước 24/07/2020.

## Giai đoạn 1 `63 parts`
* [x] 11.00 [Optimization Algorithms Index](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `01 part`
* [x] 11.01. [Optimization and Deep Learning](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+optimization-intro_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `04 part`
* [x] 11.02. [Convexity](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+convexity_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `06 part`
* [x] 11.03. [Gradient Descent](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+gd_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `09 part`
* [x] 11.04. [Stochastic Gradient Descent](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+sgd_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `07 part`
* [x] 11.05. [Minibatch Stochastic Gradient Descent](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+minibatch-sgd_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `07 part`
* [x] 11.06. [Momentum](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+momentum_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `08 part`
* [x] 11.07. [Adagrad](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+adagrad_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `05 part`
* [x] 11.08. [RMSProp](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+rmsprop_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `03 part`
* [x] 11.09. [Adadelta](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+adadelta_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `02 part`
* [x] 11.10. [Adam](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+adam_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `04 part`
* [x] 11.11. [Learning Rate Scheduling](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+lr-scheduler_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+optimization%22) `07 part`

## Giai đoạn 2 `27 parts`
* [x] 11.00 [Optimization Algorithms Index](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `01 part`
* [x] 11.01. [Optimization and Deep Learning](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+optimization-intro_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `02 part`
* [x] 11.02. [Convexity](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+convexity_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `03 part`
* [x] 11.03. [Gradient Descent](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+gd_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `03 part`
* [x] 11.04. [Stochastic Gradient Descent](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+sgd_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `03 part`
* [x] 11.05. [Minibatch Stochastic Gradient Descent](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+minibatch-sgd_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `03 part`
* [x] 11.06. [Momentum](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+momentum_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `04 part`
* [x] 11.07. [Adagrad](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+adagrad_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `02 part`
* [x] 11.08. [RMSProp](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+rmsprop_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `01 part`
* [x] 11.09. [Adadelta](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+adadelta_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `01 part`
* [x] 11.10. [Adam](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+adam_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `02 part`
* [x] 11.11. [Learning Rate Scheduling](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+lr-scheduler_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+optimization%22) `02 part`
